evalml/tests/automl_tests/test_automl_search_classification.py

```diff
@@ -594,27 +594,13 @@ def test_automl_allowed_pipelines_search(mock_fit, mock_score, dummy_binary_pipe
     assert start_iteration_callback.call_args_list[1][0][0] == dummy_binary_pipeline_class
 
 
-def test_categorical_hyperparam(X_y_multi):
```

Comment:
Weird test, Simple Imputer isn't used and the hyperparameter ranges addressed that.